collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -265,6 +265,7 @@ public void onClick(View v) {
         });
 
         mBackButton = (ImageButton) findViewById(R.id.form_back_button);
+        mBackButton.setEnabled(false);
```

Comment:
No, you are right. 
Actually, without this when we edit a saved form the prev button is enabled on the first question. Digging deeper i found that mBackButton.isShown() & mNextButton.isShown() both are 'false' when we edit saved form but are 'true' when we fill blank form. Any idea why this is so or where should i look next ?